torchbenchmark/models/llama_v2/requirements.txt

```diff
@@ -0,0 +1,3 @@
+sentencepiece
+datasets
+transformers==4.31.0
```

Comment:
This file is not needed -  we want to pin transformers version in the top-level requirements.txt file: https://github.com/pytorch/benchmark/blob/main/requirements.txt#L13